resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -1074,31 +1074,31 @@ def test_rnaseqc_qc(self):
                 },
             )
 
-        inputs = {
+        inputs_ensembl = {
             "annotation": annotation_ensembl.id,
             "alignment": alignment.id,
-            "options": {
+            "strand_detection_options": {
                 "stranded": "auto",
                 "cdna_index": salmon_index.id,
             },
         }
 
-        rnaseqc_report = self.run_process("rnaseqc-qc", inputs)
+        rnaseqc_report = self.run_process("rnaseqc-qc", inputs_ensembl)
 
         self.assertFile(
             rnaseqc_report, "metrics", outputs / "chr1_region.bam.metrics_ENSEMBL.tsv"
         )
 
-        inputs = {
-            "annotation": annotation_UCSC.id,
+        inputs_ucsc = {
+            "annotation": annotation_ucsc.id,
             "alignment": alignment.id,
-            "options": {
-                "stranded": "auto",
```

Comment:
Specify  strandedness manually, omit strand detection option and keep the detection threshold option.